Functions/Example/FirebaseFunctions.xcodeproj/project.pbxproj

```diff
@@ -494,13 +500,8 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				GCC_PRECOMPILE_PREFIX_HEADER = NO;
```

Comment:
Why this change?

Reply:
This has been the default for a very long time.